docs/user_guide.md

```diff
@@ -8,42 +8,42 @@ The initiative aims to promote a free and open internet.
 ## Introduction
 
 The platform is divided into thematic categories called magazines. By default, any user can create their own magazine
-and automatically become its owner. Then they receive a number of administrative tools that will help them personalize
+and automatically become its owner. Then they receive many administrative tools that will help them personalize
```

Comment:
This is not necessarily the same meaning. "Many" implies that the number of tools is large, while "a number of" generally carries the connotation of "maybe many, but at least several"